Tests/Api/DocumentsControllerTests.cs

```diff
@@ -0,0 +1,65 @@
+using Api.Controllers;
+using Application;
+using Application.Services;
+using Core;
+using Microsoft.Extensions.Logging;
+using Moq;
+using Xunit;
+
+namespace Tests;
+
+public class DocumentsControllerTests
+{
+    private readonly Mock<IInnerCircleHttpClient> _httpClientMock = new();
+    private readonly Mock<IPayslipsValidator> _payslipsValidatorMock = new();
+    private readonly IPayslipsValidator _validator = new PayslipsValidator();
+    private readonly ILogger<DocumentsController> _logger;
+    private readonly DocumentsController _controller;
+
+    public DocumentsControllerTests()
+    {
+        var loggerFactory = LoggerFactory.Create(builder => builder.AddConsole());
+        _logger = loggerFactory.CreateLogger<DocumentsController>();
+
+        _controller = new DocumentsController(_httpClientMock.Object, _payslipsValidatorMock.Object, _logger);
+    }
+
+    [Fact]
+    public async Task SendMailingPayslips_WhenEmailSenderServiceIsNotAvailable_CatchException()
```

Comment:
Remembered that last part of method name should be what we expect, so rename it like "ShouldThrowException". Please do that for other test methods